### ISSUE TITLE
Deprecation Warning : WARN: method access to node attributes (node.foo.bar) is deprecated and will be removed in Chef 13, please use bracket syntax (node["foo"]["bar"]) 

### ISSUE THREAD

**Issue by mberlanda** [ajgon/opsworks_ruby#89]
Dear @ajgon ,

I checked deployment log in Opsworks today and I noticed few deprecations warning regarding the way the nodes are accessed.

```
[2017-06-21T12:11:33+00:00] WARN: method access to node attributes (node.foo.bar) is deprecated and will be removed in Chef 13, please use bracket syntax (node["foo"]["bar"]) (CHEF-4)/var/chef/runs/605d8a24-b80f-4a53-a87a-48492c900115/local-mode-cache/cache/cookbooks/nginx/attributes/default.rb:43:in `from_file'.
Please see https://docs.chef.io/deprecations_attributes.html for further details and information on how to correct this problem. at /opt/chef/embedded/lib/ruby/gems/2.3.0/gems/chef-12.18.31/lib/chef/event_dispatch/dispatcher.rb:43:in `call'
[2017-06-21T12:11:33+00:00] WARN: Chef::Provider::AptRepository already exists!  Cannot create deprecation class for LWRP provider apt_repository from cookbook apt
[2017-06-21T12:11:33+00:00] WARN: AptRepository already exists!  Deprecation class overwrites Custom resource apt_repository from cookbook apt
[2017-06-21T12:11:33+00:00] WARN: Chef::Provider::YumRepository already exists!  Cannot create deprecation class for LWRP provider yum_repository from cookbook yum
[2017-06-21T12:11:33+00:00] WARN: YumRepository already exists!  Deprecation class overwrites Custom resource yum_repository from cookbook yum
```
I know that this solution is designed to work in Chef 12, but it could be interesting to fix as [described here](https://docs.chef.io/deprecations_attributes.html).

I would have put a low priority flag but I haven't found any

**Comment by ajgon** [comment]
Hey,

The problem is, it's not the call from `opsworks_ruby` cookbook itself, but from `nginx`. `opsworks_ruby` always uses bracket syntax IIRC. 

However - which version of cookbook do you use? `nginx` dependency is deprecated and was replaced by `chef_nginx` some time ago.

**Comment by mberlanda** [comment]
Hey,
I just realized that you published several version since the one I forked - November 2 commit (a55d317be77d955f32d9a96d2c562da3dd88609d).

Since then I made a custom recipe to update the schedule of [`whenever`](https://github.com/javan/whenever) gem after deploy.

I have sync my repo with your latest version and everything seems to work fine (except for a missing gem 
> Post-install message from minitar:
> The `minitar` executable is no longer bundled with `minitar`. If you are
> expecting this executable, make sure you also install `minitar-cli`.

I updated the custom cookbook on some projects already and everything seems to work perfectly. I still get this message in the log but it's just an info
```
[2017-07-17T09:54:16+00:00] INFO: HTTP Request Returned 404 Not Found: Object not found: chefzero://localhost:8889/nodes/{instance-name}
```
